File: MODUL 5/PRAK504-2410817210005-Muhammad_Irgi_Fahrezha.py
def reverse (angka):
    balik = 0
    
    while angka != 0:
        sisa = angka % 10
        balik = balik * 10 + sisa
        angka //= 10

    return balik

A, B = map(int, input().split())

A = reverse(A)
B = reverse(B)

C = A + B
print(reverse(C))

File: MODUL 5/PRAK504-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int reverse(int angka) {
    int balik = 0, sisa;

    while (angka != 0) {
        sisa = angka % 10;
        balik = balik * 10 + sisa;
        angka /= 10;
    }
    return balik;
}

int main () {
    int A, B;
    scanf("%d %d",&A,&B);
    A = reverse(A);
    B = reverse(B);
    int C = A+B;
    printf("%d",reverse(C));

    return 0;
}